### PR TITLE
Ticket 9

### PR DESCRIPTION
**Pull request by MSalim16**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -23,7 +24,8 @@ const router = createBrowserRouter(
     <Route path="/" element={<NavBar />}>
       <Route index element={<Home />} />
       <Route path="reviews" element={<Review />} />
-      <Route path="reviews/:review_id" element={<SingleReview />} />
+      <Route path="reviews/:category" element={<Review />} />
```
Comment (P-Copley) [inline src/App.js:27]: If you're going to a parametric it should generally be a subsection of the  previous path. i.e. `/categories/:category`.

This avoids conflicts with the below route that starts with `/reviews/review` (which can just be `/reviews/:review_id`)

File: src/api.js [hunk 2]
```diff
@@ -4,20 +4,24 @@ const gamesUrl = axios.create({
   baseURL: "https://mohamed-salims-games-api.onrender.com/api",
 });
 
-export const getReviews = () => {
-  return gamesUrl.get("/reviews").then((reviews) => {
+export const getReviews = category => {
+  let params;
+  category === "all-reviews"
+    ? (params = undefined)
+    : (params = { params: { category } });
```
Comment (P-Copley) [inline src/api.js:11]: You shouldn't do side-effects in ternaries, that's what if statements are for. 

This should be:
```js
let params = category === "all-reviews" ? undefined : { params: { category } }
```

A ternary is designed to evaluate to a value and you should use that value. 👍

File: src/App.js (hunk 1 above)
Comment (MSalim16) [reply src/App.js:27]: ok thanks will implement that now That actually caused me a lot of stress yesterday as the links were similar it stopped the functionality of whichever was underneath the other in the app

File: src/api.js [hunk 3]
```diff
@@ -39,7 +42,14 @@ export const postComment = (user, review_id, commentBody) => {
...
     })
-    .then((comment) => {
+    .then(comment => {
       return comment.data.comment;
     });
 };
+
+export const getCategories = () => {
+  return gamesUrl.get("/categories").then(categories => {
+    console.log(categories.data.categories);
```
Comment (P-Copley) [inline src/api.js:52]: This log's snuck in, remove it please 👍

File: src/api.js (hunk 2 above)
Comment (MSalim16) [reply src/api.js:11]: thx for that

File: src/api.js (hunk 3 above)
Comment (MSalim16) [reply src/api.js:52]: done

**Pull request merged**